packages/cadl-rlc-emitter/test/integration/generated/clients/interfaces/typescript.json

```diff
@@ -7,5 +7,6 @@
     "description": "Client Interface Test Service",
     "version": "1.0.0"
   },
-  "isCadlTest": true
+  "isCadlTest": true,
+  "overrideTitle": "InterfaceClient"
```

Comment:
I don't think we should have override here, just put title here. and then use it to override whatever we defined in the cadl. the same as what we do in typescript.md.